src/factories/configurationContext.ts

```diff
@@ -2,7 +2,7 @@ import { Factory } from "fishery"
 import { Configuration } from "../ConfigurationContext"
 
 export default Factory.define<Configuration>(() => ({
-  appDownloadLink: "appDownloadLink",
+  appDownloadUrl: "appDownloadUrl",
```

Comment:
changed Link to Url to be consistent with the rest of the provided url variable names